js/config.js

```diff
@@ -63,6 +63,11 @@ export const title = _meta('site-title');
 export const csrf_token = _meta('csrf-token');
 
 /**
+ * Is the check url feature enabled ?
+ */
+export const check_urls = JSON.parse(_meta('check-urls'));
```

Comment:
Oh nice, I didn't think about using `JSON.parse` for that in territories, I'll update :)